src/posts/paper/13-q-and-a.ts

```diff
@@ -179,6 +179,48 @@ However, we have retracted this method for the following reasons:
+
+People who have been vaccinated are less likely to catch COVID and transmit it to others, which results in them getting fewer microCOVIDs from activities.
+However, they are not totally immune to COVID, and it is quite easy for a vaccinated person to counteract the decreased risk per activity by doing lots of
+risky activities (for instance, the Moderna vaccine confers a 1/5x multiplier for incoming microCOVIDs, so a vaccinated person who does 5x more risky behavior
+after being vaccinated would be just as many microCOVIDs as before).
+
+There are two questions that are important for understanding vaccines:
+
+1. How much less likely is it for a vaccinated individual to catch COVID? This determines the reduction in microCOVIDs a vaccinated individual receives from a given action.
+2. If a vaccinated individual contracts COVID, how much less (or more) likely is this to result in negative consequences (hospitalizaton or death). This determines how much a person can increase their microCOVID budget for being vaccinated (on top of being able to do more activities from part 1).
+
+### 1. How much less likely is it for vaccinated individuals to catch COVID? (Reduction in microCOVIDs)
+
+We do not know for sure how likely it is for a vaccinated individual to catch to COVID. Vaccinated individuals still have a chance of becoming sick (even if they are not seriously ill), and may be asymptomatic carriers of the virus, capable of infecting others without having symptoms themselves. None of the vaccine studies have directly addressed what the chances of transmission from a vaccinated individual are. We attempt to make educated guesses by pulling in other sources. For our derivation of these numbers, see the [Research Sources](/paper/14-research-sources#vaccines).
+
+|             | microCOVID multiplier 7 days after 1st dose | microCOVID multiplier 14 days after 2nd dose |
+| ----------- | ------------------------------------------- | -------------------------------------------- |
+| AstraZeneca | 0.56                                        | 0.4                                          |
+| Moderna     | 0.56                                        | 0.2 [^moderna]                               |
+| Pfizer [^pfizer] | 0.56                                   | 0.2                                          |
+
+For perspective, the first dose of either vaccine confers equivalent protection as wearing a good quality cloth mask all day, every day. Completing the 2nd dose of AstraZeneca’s vaccine confers more protection than wearing a surgical mask 24/7, and Pfizer or Moderna’s vaccine is as protective as wearing a KN95 mask all the time.
```

Comment:
I would explicitly clarify that "all day every day" means: "... even when you're with people you live with."

The first time I read it, I thought "yeah, I already do wear a KN95 all the time." I had to re-read it to get that you meant literally all the time, even when I'm with my housemates. I know it should be obvious by how you wrote it, bu the nuance slipped passed me on the first read.

Reply:
Here's another "take it or leave it" suggestion. Here's how I'd write it. 

"You can think of being fully vaccinated as giving you the equivalent level of protection as wearing a good mask all day, every day (even when you're around people you live with or see regularly). Completing the second dose of Pfizer or Moderna’s vaccine is as protective as though you were permanently wearing a KN95 mask, whereas AstraZeneca’s would be as though you were permanently wearing a surgical mask."

Reply:
Ooh, this sounds like it is very much not communicating what I want to get across. 

I want to say "this is an equivalent multiplier to a kn95 mask" and very much not "this replaces wearing a mask"

Try this edit?

Reply:
Oh yeah I didn't mean to imply it replaced a mask. I'll take a look at your edits.